dashboard/fixtures/certificates.json

```diff
@@ -5,14 +5,14 @@
 	    "certificate_type": "verified",
 	    "status": "downloadable",
 	    "download_url": "http://www.example.com/demo.pdf",
-	    "grade": "0.98"
+	    "grade": 98
```

Comment:
@noisecapella - actually, im noticing something pretty weird here. that edx post-processing code that i mentioned earlier doesn't even seem to be in this branch. i'll dig into this a little more